x/auth/ante/validator_tx_fee.go

```diff
@@ -7,9 +7,14 @@ import (
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 )
 
-// checkTxFeeWithValidatorMinGasPrices implements the default fee logic, where the minimum price per
+// DefaultPriorityReduction is the default amount of price values required for 1 unit of priority.
+// Because priority is `int64` while price is `big.Int`, it's necessary to scale down the range to keep it more pratical.
+// The default value is the same as the `sdk.DefaultPowerReduction`.
+var DefaultPriorityReduction = sdk.DefaultPowerReduction
```

Comment:
Should it default to `1`?

Reply:
I tend to sort of agree here. Why not just write your own decorator? The one in the SDK is meant to be extremely basic.

Reply:
I'm trying to use the default one as a fallback, while the custom one can be enabled with a parameter.